forum/investors/models.py

```diff
@@ -52,4 +56,34 @@ class Meta:
         verbose_name_plural = 'Investors'
 
     def __str__(self):
-        return self.company_name
\ No newline at end of file
+        return self.company_name
+
+
+class InvestorStartup(models.Model):
+    """
+    Represents a many-to-many relationship between investors and startups, 
+    indicating which investors are following which startups.
+
+    Attributes:
+        investor_id (ForeignKey): A reference to the User model, representing 
+        the investor who follows a startup. The relationship is set to cascade 
+        on deletion, meaning if the user is deleted, their follows will also be removed.
+
+        startup_id (ForeignKey): A reference to the Startup model, representing 
+        the startup being followed by the investor. Like the investor, this relationship 
+        also cascades on deletion.
+
+        saved_at (DateTimeField): A timestamp indicating when the investor 
+        started following the startup. This field is automatically set to 
+        the current date and time when the relationship is created.
+    """
+
+    investor_id = models.ForeignKey(User, on_delete=models.CASCADE, related_name='investor_startups')
+    startup_id = models.ForeignKey(Startup, on_delete=models.CASCADE, related_name='startup_investors')
```

Comment:
It’s common practice to use `investor` and `startup` (without `_id`) as the field names in Django models for `ForeignKey` fields. The `_id` suffix is automatically created by Django at the database level.